components/MapScreen.tsx

```diff
@@ -1,25 +1,33 @@
-import "react-native-gesture-handler";
-import React, { useState, useEffect } from "react";
+import 'react-native-gesture-handler';
+import React, { useState, useEffect } from 'react';
 import {
   StyleSheet,
   View,
   Dimensions,
   TouchableOpacity,
   Text,
-} from "react-native";
-import MapView, { Marker } from "react-native-maps";
-import { getHospitals } from "../api";
+} from 'react-native';
+import MapView, { Marker } from 'react-native-maps';
+import { getHospitals } from '../api';
 
 export default function MapScreen({ navigation }) {
   const [hospitals, setHospitals] = useState([]);
 
   useEffect(() => {
-    getHospitals().then(setHospitals);
-  });
+    getHospitals().then((hospitals) => {
+      setHospitals(hospitals);
+    });
+  }, []);
 
   return (
     <View style={styles.container}>
-      <MapView style={styles.map}>
+      <MapView
+        provider={'google'}
+        style={styles.map}
+        showsUserLocation={true}
+        showsMyLocationButton={true}
+        showsCompass={true}
```

Comment:
In react, for boolean properties, you don't need to specify the `true` value :) 
```suggestion
        showsUserLocation
        showsMyLocationButton
        showsCompass
```